flight/mcu/lib/teensyShared/ActiveControl.cpp

```diff
@@ -1,73 +1,84 @@
 #include "ActiveControl.h"
 
-#include "PWMServo.h"
-#include "ServoControl.h"
-#include "acShared.h"
-#include "dataLog.h"
-#include "sensors.h"
-
-ActiveControl::ActiveControl(struct pointers* pointer_struct, PWMServo* ccw,
-                             PWMServo* cw)
-    : activeControlServos(ccw, cw) {
-    gy = &pointer_struct->sensorDataPointer->lowG_data.gy;
+#include "rocketFSM.h"
+#include "thresholds.h"
+
+Controller::Controller(struct pointers* pointer_struct, PWMServo* controller_servo)
+    : activeControlServos(controller_servo) {
+    controller_servo_ = controller_servo;
+    stateData_ = &pointer_struct->sensorDataPointer->state_data;
     current_state =
         &pointer_struct->sensorDataPointer->rocketState_data.rocketState;
-    mutex_lowG_ = &pointer_struct->dataloggerTHDVarsPointer.dataMutex_lowG;
+    uint32_t* ac_coast_timer = &pointer_struct->rocketTimers.coast_timer;
```

Comment:
this ac_coast_timer is never used, did you mean the member variable ac_coast_timer?

Reply:
@gautamdayal this one is a you thing I think?